src/libs/table/TBody.tsx

```diff
@@ -34,7 +34,7 @@ export const TBody = <D extends object>({
               'h-64',
               '!text-white/80',
               'text-16',
-              'hover:bg-darkSilver',
+              'hover:bg-neutral-800',
```

Comment:
darkSilver not defined in the issue #1034 

neutral-900 might be closer to it than neutral-800